extensions/ql-vscode/src/pure/interface-types.ts

```diff
@@ -189,7 +190,8 @@ export type FromResultsViewMsg =
   | ViewLoadedMsg
   | ChangePage
   | OpenFileMsg
-  | TelemetryMessage;
+  | TelemetryMessage
+  | UnhandledErrorMessage;
```

Comment:
Should we perhaps be creating a separate type which contains all "common" messages. Currently this would be the `ViewLoadedMsg`, `TelemetryMessage`, and `UnhandledErrorMessage`. If we add a new view, these all need to be included, otherwise the extension might crash if any of these messages are sent anyway.

Reply:
Sounds good to me. Making a type for all the "command" messages should be fine, and then we include that type in all the types like `FromVariantAnalysisMessage`. Earlier when I first started this PR I tried to get `AbstractWebview` to handle these common messages automatically, but that proved too difficult and I couldn't get the types to work out. So each view still has to implement these messages and remember to include them in it's messages type, but at least by grouping them together it should be harder to miss one.